### DISCUSSION TITLE
[론칭 페스티벌] 4차 스프린트 및 론칭 페스티벌 회고

### DISCUSSION THREAD

**Discussion by Minjoo522** [woowacourse-teams/2024-devel-up#468] [백로그 논의]
#### 논의가 필요한 내용

4차 스프린트 및 론칭 페스티벌 회고를 남겨주세요.
자세하게 남길수록 다음 스프린트 때 더 많은 도움이 될 것 같아요!

**Comment by robinjoon** [comment]
![20240823_162713.jpg](https://github.com/user-attachments/assets/1e538579-a27d-48cd-8e27-2a2a28d9b615)
